test/test_shared_library.py

```diff
@@ -0,0 +1,98 @@
+    test_schema= json.loads(open('translator/sample files/validation_schema/wzdx_v3.0_feed.json').read())
+    validate_write=translator_shared_library.validate_wzdx(valid_wzdx_data,test_schema)
+    assert validate_write == True
+
+def test_validate_write_invvalid_wzdx_data():
```

Comment:
Spelling issue - only one 'v' in invalid.

Reply:
should be fixed now